### PR TITLE
remove src files for code review

### PR DESCRIPTION
**Pull request by ZacharyStair**
this code review is backwards because if i change the base (i.e. merge 'master' into 'empty'), 'empty' has all of the commits from master so theres nothing to merge.

You can just leave comments on the red parts and i'll do my best to keep this pr up to date by rebasing onto master

### CODE REVIEW

**Comment by ZacharyStair** [comment]
@danidimitrova @justinpurcell1 could you review this?

**Review by justinpurcell1 — APPROVED** [review]
everything i understand i am ok with

File: src/google.js [hunk 1]
```diff
@@ -1,165 +0,0 @@
...
-              null,
-              Object.assign({}, userState, {
-                actionType: args[0],
-                action,
-                tokens: null
-              }),
-              () => this.reauth(action, userId)
-            );
-        });
-        res.status(201).end();
```
Comment (testradav) [inline src/google.js]: 200?

File: src/app.js [hunk 2]
```diff
@@ -1,179 +0,0 @@
...
- * @param {PouchDB} store
- * @param {Function<String>} wwToken - returns a valid WW token to post as the app to WW
- */
-export const messagesCallback = (appId, store, wwToken) =>
-  (req, res) => {
-    log('Received body %o', req.body);
-
-    // Respond to the Webhook right away, as any response messages will
-    // be sent asynchronously
-    res.status(201).end();
```
Comment (testradav) [inline src/app.js]: 200?

File: src/messages.js [hunk 3]
```diff
@@ -1,160 +0,0 @@
...
-          title: title,
-          text: text,
-
-          actor: {
-            name: actor
-          }
-        }]
-      }
-    }, (err, res) => {
-      if (err || res.statusCode !== 201) {
```
Comment (testradav) [inline src/messages.js:147]: `<200 || >299`?

**Review by testradav — COMMENTED** [review]
this is the message in gmail after the has been allowed access....  not sure where the `Hellow World` words are coming from... did not find them in the code here
image

also this is how it shows in connected apps of gmail:
image

**Comment by ZacharyStair** [comment]
@testradav 'hello world' is the name of the app that I created on the google cloud platform. I guess if other people are going to be using this app i should probably change that...

Also the changes i'm pushing to origin aren't reflected in this pr, i'll rebase it now

File: src/messages.js (hunk 3 above)
Comment (ZacharyStair) [reply src/messages.js:147]: We know that a successfully sent message will return a 201 so we might as well just check that. I don't foresee that interface changing anytime soon, and even if it does, not much in the project relies on the message callback

**Review by testradav — APPROVED** [review]
put through @miguelestrada for review

**Pull request closed without merging**